addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -411,7 +411,10 @@ def attributes(self, record, field_name, options, values=None):
                         # group allowing to bypass the sanitation saved that
                         # field previously. Mark the field as not editable.
                         attrs['data-oe-sanitize-prevent-edition'] = 1
-                if not (field.sanitize_overridable and record.user_has_groups('base.group_sanitize_override')):
```

Comment:
@rdeodoo Working on a third version, here are tables for each version <https://docs.google.com/spreadsheets/d/1r0V4QSnQ2X5YwTa_e4kGDlAdGTArmG3D58ZfoVV359c/edit#gid=1571669564>.
IMO:
- In the current commit version, the "was prevented" condition makes no sense, it can just be removed.
- In your suggestion: overriders are able to bypass the "no_block" rule

I think I will take your version, but adapt for the "no_block" if `sanitize_attributes` before the return shortcut.
Or maybe you think they should be able to bypass it too ?

Reply:
Nobody should face the issue anymore once the forward chain of https://github.com/odoo/odoo/pull/119543 is complete.

Reply:
Ultimately ended up applying your suggestion (just adjusted a quote typo, and returning `attrs` in each `return`)